src/etc/test-float-parse/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(f16)]
```

Comment:
Adding this will require changing

https://github.com/rust-lang/rust/blob/8c392966a013fd8a09e6b78b3c8d6e442bc278e1/src/bootstrap/src/core/build_steps/tool.rs#L1243

to allow `f16` btw.